### PR TITLE
NR-121907: use release toolkit

### PR DESCRIPTION
**Pull request by sigilioso**
- [x] The branch of the PR is prefixed with renovate/*
- [x] Tests pass in renovate branches
- [x] security workflow is included and will pass in renovate branches
- [x] the changelog is validated in push_pr workflow
- [x] The repository has the secrets available
- [x] Release schedule is documented and not colliding
- [x] rt-update-changelog is in Makefile
- [x] Tags with leading v are safe
- [x] Common renovate config is pointed
- [x] Potential troublesome dependencies are checked
- [x] pre-release and release workflows are updated to notify failures
- [x] CHANGELOG.partial.md is in .gitignore
- [x] release-toolkit action excluded/included fields are set
- [x] Docker image pings go patch version
- [ ] The dependency dashboard is sanitized --> It will be enabled when the PR is merged, and then we need to remove Worktato comment and set `dependencies` label.

### CODE REVIEW

File: .github/workflows/on_prerelease.yml [hunk 1]
```diff
@@ -82,18 +82,6 @@ jobs:
...
           GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
           GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
-      - name: Notify failure via Slack
-        if: ${{ failure() }}
-        uses: archive/github-actions-slack@master
-        with:
-          slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
-          slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
-          slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
-      - name: Test package installability
```
Comment (paologallinaharbur) [inline .github/workflows/on_prerelease.yml:92]: did you remove this intentionally?

Comment (sigilioso) [reply .github/workflows/on_prerelease.yml:92]: No! 😓 Thanks for noticing!

File: .github/workflows/on_release.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Release pipeline
+name: Create release artifacts
```
Comment (paologallinaharbur) [inline .github/workflows/on_release.yml:1]: in this file you forgot the notification

File: .github/workflows/on_prerelease.yml (hunk 1 above)
Comment (sigilioso) [reply .github/workflows/on_prerelease.yml:92]: Restored here: 57dd2a515d1e9211b77af700641bfb33f05c5b99

**Review by paologallinaharbur — APPROVED** [review]
Feel free to merge it once the two issues are solved 😄

**Pull request merged**